### PR TITLE
Improve documentation for meta tags

### LINKED ISSUE #550: New 1.0.0 Install, No Meta Tags Rendered (meta is empty)
# Goal

1. Primarily, only one feature from this plugin:
    - "Twitter cards, Open Graph and Google+ snippets meta tags"
2. Possibly, other features from this plugin.

# Problem

1. The `{% include "meta/meta.html" %}` (in my `base.html`) does not render anything.
2. The `meta` variable is empty. _Tested via:_
    - `{% debug %}` in my site's `base.html` template
    - `{% debug %}` in `/usr/local/lib/python3.6/site-packages/meta/templates/meta/meta.html`
3. I am new to much of Python, Django, and Django CMS.

# Notes

- This plugin supports the desired feature via [`django-meta`, which does not officially support Django 3.1+](https://github.com/nephila/django-meta#django).
- All other basic blog features seem to be working.

# Environment

- OS Name: Linux _(via [Docker Image `python:3.6-buster`](https://hub.docker.com/_/python?tab=tags&page=3))_
- OS Version: _Unknown_
- Python Version: `3.6.10`
- Django CMS Version: `3.6.0`

# Installation

## `requirements.txt`
```
...
djangocms-blog==1.0.0
...
```

## `settings.py`
```
    ...
    'filer',
    'easy_thumbnails',
    …

    # Support `djangocms-blog`
    # 'filer',              # Already added
    # 'easy_thumbnails',    # Already added
    'aldryn_apphooks_config',
    'parler',
    'taggit',
    'taggit_autosuggest',
    'meta',
    'sortedm2m',
    'djangocms_blog',

    ...
```

## `urls.py`
```
...

# Support `djangocms-blog`
urlpatterns += [
    url(r'^taggit_autosuggest/', include('taggit_autosuggest.urls')),
]

...
```

## `base.html`
```
{% load cms_tags staticfiles menu_tags sekizai_tags meta cache i18n %}
<!doctype html>
<html>

<head {% meta_namespaces %}>
  <title>{% block title %}This is my new project home page{% endblock title %}</title>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
  {% include "meta/meta.html" %}

  <link rel="icon" href="{% static 'site_cms/images/favicons/favicon.ico' %}" type="image/x-icon" />
  ...
</head>
...
```

# Runtime

## `{% debug %}`

This is just the beginning of the output from a test blog page.
```
{'cms_edit_off': 'edit_off', 'cms_edit_on': 'edit', 'cms_renderer': , 'cms_structure_on': 'structure', 'cms_toolbar': , 'cms_version': '3.6.0', 'django_version': '1.11.22', 'login_form': , 'python_version': '3.6.10'}{'instant_article': False, 'meta': , 'object': , 'post': , 'use_placeholder': True, 'view': }{}{'CMS_MEDIA_URL': '/media/cms/', 'CMS_TEMPLATE': . at 0x7faf58478c80> …
```

## Blog Config

- It is the default automatic entry.
- I have added useless values to the "Open Graph", "Twitter", and "Google+" entries, like `FB_APP_ID`, `TW_AUTHOR_HANDLE`, etc.

## Blog Category/Post

- Inside a solitary test category with descriptive test text for all text entry fields.
- A default blog page with descriptive test text for all text entry fields.

### PR DESCRIPTION
**Pull request by yakky**
Fix #550

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -76,6 +76,12 @@ suited for your deployment.
     META_SITE_PROTOCOL = 'http'
     META_USE_SITES = True
 
+* For meta tags support enable the needed types::
```
Comment (tacc-wbomar) [inline docs/installation.rst:79]: Extra colon i.e. `::` instead of `:`.

Comment (yakky) [reply docs/installation.rst:79]: the `::` is used to render the following lines with code formatting (see similar formatting in the lines above)

Comment (tacc-wbomar) [reply docs/installation.rst:79]: Ah, okay. I've been working with Markdown all weekend, and didn't notice that this `.rst`—not my realm 😅.

Comment (yakky) [reply docs/installation.rst:79]: :joy:

**Pull request merged**